frontend/src/employee-frontend/components/UnitPage.tsx

```diff
@@ -103,22 +103,19 @@ const UnitPage = React.memo(function UnitPage() {
       unitInformation.value.permittedActions.has('READ_DETAILED')
         ? [
             {
-              id: 'waiting-confirmation',
-              link: `/units/${id}/waiting-confirmation`,
-              label: i18n.unit.tabs.waitingConfirmation
-            },
-            {
-              id: 'placement-proposals',
-              link: `/units/${id}/placement-proposals`,
-              label: i18n.unit.tabs.placementProposals,
-              counter: unitData
-                .map((data) => data.placementProposals?.length)
-                .getOrElse(undefined)
-            },
-            {
-              id: 'applications',
-              link: `/units/${id}/applications`,
-              label: i18n.unit.tabs.applications
+              id: 'application-process',
+              link: `/units/${id}/application-process`,
+              label: i18n.unit.tabs.applicationProcess,
+              counter: unitData.isSuccess
+                ? unitData
+                    .map(
+                      (data) =>
+                        (data.applications?.length ?? 0) +
+                        (data.placementPlans?.length ?? 0) +
+                        (data.placementProposals?.length ?? 0)
+                    )
+                    .getOrElse(0)
+                : 0
```

Comment:
You don't need to test for `unitData.isSuccess` when using `unitData.map`. The idea of `Result.map` is that you can map the `Success` state but for other states it doesn't have any effect. The idea of `getOrElse` is that if the value is not `Success`, you can fall back to some value.